src/main.js

```diff
@@ -1 +1,10 @@
-console.log("Car");
+class Car {
+  constructor({ name }) {
+    this.name = name;
+    this.position = 0;
+  }
```

Comment:
```suggestion
class Car {
  position = 0;
  
  constructor({ name }) {
    this.name = name;
  }
```

이렇게 class field에 선언하는 방법도 있답니다!